src/libostree/ostree-kernel-args.c

```diff
@@ -829,3 +829,27 @@ ostree_kernel_args_append_if_missing (OstreeKernelArgs  *kargs,
  
   ostree_kernel_args_append (kargs, arg);
 }
+
+/**
+ * ostree_kernel_args_append_contains:
+ * @kargs: a OstreeKernelArgs instance
+ * @arg: key or key/value pair to check
+ *
+ * Search for @arg which is in the form of key=value pair at the hash table kargs->table
+ * and returns true if finds it.
```

Comment:
Also, use `%TRUE` and since this function returns a value this should probably have a line like:

```
Returns: %TRUE if @arg is contained in @kargs, %FALSE otherwise
```